hyperspy/tests/signal/test_eds_tem.py

```diff
@@ -135,7 +135,7 @@ def test_get_calibration_from(self):
 class Test_quantification:
 
     def setUp(self):
-        s = EDSTEMSpectrum(np.ones([2, 1024]))
+        s = EDSTEMSpectrum(np.ones([2, 2, 1024]))
         energy_axis = s.axes_manager.signal_axes[0]
```

Comment:
I'm not sure if I think you actually need to change anything - my gut is that the way it is implemented is ok. The key thing is to make sure it is really really obvious in the documentation that these things have to be in order - its a much more strict requirement than for other methods.

It's just to remember that although these things and sub-pixel scanning should be well known for those on the technique development side, many HyperSpy users will be doing just a bit of relatively routine characterisation with good analysis and I really don't think we can assume that people will firstly be able to do sub-pixel scanning on their machine, and secondly appreciate why it matters here -- so we have to point it out.